other/webpack.config.es6.js

```diff
@@ -185,7 +184,7 @@ const config = {
     formatter: require('eslint-friendly-formatter'),
   },
 
-  devtool: 'source-map',
+  devtool: 'cheap-module-eval-source-map',
```

Comment:
Yeah, that's a good catch. Basically we need a ternary operator here. I misread one of the tables on webpack's "stellar" documentation to mean that this option was ignored on prod builds, but I never realized that webpack actually doesn't know what a prod build is.

I'll send that fix in tomorrow.